src/components/AppMap.vue

```diff
@@ -186,6 +252,30 @@ class AppMapAutoUpdator {
     this.enabled = false
   }
 };
+
+function getCookie (cookieName) {
+  var name = cookieName + '='
+  var decodedCookie = decodeURIComponent(document.cookie)
+  var cookies = decodedCookie.split(';')
+  for (var i = 0; i < cookies.length; i++) {
+    var cookie = cookies[i]
+    while (cookie.charAt(0) === ' ') {
+      cookie = cookie.substring(1)
+    }
+    if (cookie.indexOf(name) === 0) {
+      return cookie.substring(name.length, cookie.length)
+    }
+  }
+  return ''
+}
+
+function setCookie (name, value, expireDays) {
+  var date = new Date()
+  date.setTime(date.getTime() + (expireDays * 24 * 60 * 60 * 1000))
+  var expires = 'expires=' + date.toUTCString()
+  document.cookie = name + '=' + value + ';' + expires + ';path=/'
+}
```

Comment:
Suggest using local storage instead of cookie. Per http://stackoverflow.com/a/3220802/4545351